dists/scummvm.6

```diff
@@ -14,226 +13,400 @@
-.It Fl -soundfont= Ns Ar FILE
-Select the SoundFont for MIDI playback (only supported by some MIDI drivers).
+Extra path to additional game data.
+.It Fl -filtering
+Force filtered graphics mode.
+.It Fl -game= Ns Ar ID
+In combination with
+.Fl -add
+or
+.Fl -detect
+only adds or attempts to detect the game with id
+.Ar ID .
+.It Fl -gui-theme= Ns Ar THEME
+Select GUI theme
+.Po
+.Ar default ,
+.Ar modern ,
+.Ar classic
+.Pc .
+.It Fl -joystick= Ns Ar NUM
+Enable joystick input (default: 0 = first joystick).
+.It Fl -iconspath= Ns Ar PATH
+Path to additional icons for the launcher grid view.
+.It Fl -list-all-debugflags
+Display list of all engine specified debugflags.
+.It Fl -list-all-engines
+Display list of all detection engines and exit.
+.It Fl -list-all-games
+Display list of all detected games and exit.
+.It Fl -list-audio-devices
+List all available audio devices.
+.It Fl -list-debugflags= Ns Ar engine
+Display list of engine specified debugflags.
+if
+.Ar engine= Ns global
+or
+.Ar engine
+is not specified, then it will list global debugflags.
+.It Fl -list-engines
+Display list of supported engines and exit.
+.It Fl -list-saves
+Display a list of saved games for the target specified with
+.Fl -game= Ns Ar TARGET
+or all targets if none is specified.
+.Fl -game= Ns Ar ID
+In combination with
+.Fl -add
+or
+.Fl -detect
+only adds or attempts to detect the game with id
+.Ar ID .
+.It Fl -list-themes
+Display list of all usable GUI themes.
+.It Fl -md5
+Shows MD5 hash of the file given by
+.Fl -md5-path= Ns Ar PATH .
+If
+.Fl -md5-length= Ns Ar NUM
+is passed then it shows the MD5 hash of the first or last
+.Ar NUM
+bytes of the file given by
+.Ar PATH .
+If
+.Fl -md5-engine= Ns Ar ENGINE_ID
+is passed, it fetches the MD5 length automatically, overriding
+.Fl -md5-length .
+.It Fl -md5mac
+Shows MD5 hash for both the resource fork and data fork of the
+mac file given by
+.Fl -md5-path= Ns Ar PATH .
+If
+.Fl -md5-length= Ns Ar NUM
+is passed then it shows the MD5 hash of the first or last
+.Ar NUM
+bytes of each fork.
+.It Fl -midi-gain= Ns Ar NUM
+Set the gain for MIDI playback, 0-1000 (default: 100).
+.br
+(only supported by some MIDI drivers)
 .It Fl -multi-midi
 Enable combination of AdLib and native MIDI.
 .It Fl -native-mt32
 True Roland MT-32 MIDI (disable GM emulation).
-.It Fl -enable-gs
-Enable Roland GS mode for MIDI playback.
+.It Fl -no-filtering
+Force unfiltered graphics mode.
+.It Fl -opl-driver= Ns Ar DRIVER
+Select AdLib (OPL) emulator
+.Po
+.Ar db ,
+.Ar mame ,
+.Ar nuked
+.Pc .
 .It Fl -output-rate= Ns Ar RATE
 Set output sample rate in Hz (e.g. 22050).
-.It Fl -opl-driver= Ns Ar DRIVER
-Select AdLib (OPL) emulator (\fIdb\fR, \fImame\fR, \fInuked\fR)
-.It Fl -aspect-ratio
-Enable aspect ratio correction.
+.It Fl -platform= Ns Ar WORD
+Specify platform of game
+.Po
+.Ar 2gs ,
+.Ar 3do ,
+.Ar acorn ,
+.Ar amiga ,
+.Ar atari ,
+.Ar c64 ,
+.Ar fmtowns ,
+.Ar mac ,
+.Ar nes ,
+.Ar pc ,
+.Ar pce ,
+.Ar segacd ,
+.Ar windows
```

Comment:
We have more platforms. The full list is at https://github.com/scummvm/scummvm/blob/master/common/platform.cpp#L28

Reply:
added a commit with the rest of the first strings in the table